app/src/main/res/layout/activity_feedback.xml

```diff
@@ -34,39 +35,78 @@
                 android:layout_height="wrap_content"
                 android:layout_centerVertical="true"
                 style="@style/AppTheme.Toolbar.Button.Flat"
+                android:contentDescription="@string/feedback_back_content_description"
                 android:src="@drawable/ic_back_arrow"/>
 
             <TextView
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
+                android:layout_toStartOf="@+id/btnSaveImage"
+                android:layout_toEndOf="@id/btnBack"
                 android:layout_centerInParent="true"
                 style="@style/AppTheme.Toolbar.TextAppearance.Title"
                 android:text="@string/feedback_title"/>
 
             <ImageButton
                 android:id="@+id/btnSaveImage"
-                android:layout_width="wrap_content" android:layout_height="wrap_content"
-                android:layout_alignParentRight="true"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentEnd="true"
                 android:layout_centerVertical="true"
                 style="@style/AppTheme.Toolbar.Button.Flat"
+                android:contentDescription="@string/feedback_save_content_description"
                 android:src="@drawable/ic_save"
                 android:text="@string/feedback_save"/>
+
         </RelativeLayout>
+
     </android.support.v7.widget.Toolbar>
 
-    <android.support.v7.widget.CardView
-        android:id="@+id/feedbackView"
+    <ScrollView
+        xmlns:android="http://schemas.android.com/apk/res/android"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/feedback_cardview_margin"
+        android:layout_below="@id/toolbar"
         android:layout_alignParentBottom="true"
-        android:alpha="0.0"
-        android:animateLayoutChanges="true"
-        card_view:cardCornerRadius="@dimen/feedback_cardview_corner_radius"
-        tools:alpha="1.0">
+        android:fillViewport="true">
+
+        <RelativeLayout
```

Comment:
Neither in the Scrollview nor in the FrameLayout. I tried both at the same time too.